diff --git a/dist/index.mjs b/dist/index.mjs
--- a/dist/index.mjs
+++ b/dist/index.mjs
@@ -7235,13 +7235,8 @@ async function pushOrResolveImage(config2, container, imageName, imageConfig, bu
       targetTag: builtImage.localTag,
       accountId,
       complianceConfig: config2,
-      cleanupSourceTag: true
+      cleanupSourceTag: false
     });
-    for (const image of builtImages) {
-      if (image.localTag === builtImage.localTag) {
-        image.localTagCleaned = true;
-      }
-    }
     return "remoteDigest" in imageRef ? imageRef.remoteDigest : imageRef.newTag;
   } catch (error62) {
     if (error62 instanceof Error) {
